ports/copypp/portfile.cmake

```
@@ -0,0 +1,25 @@
+set(VCPKG_BUILD_TYPE release)
```

```suggestion
set(VCPKG_BUILD_TYPE release) # header-only
```
Keep the comment, it is helpful for reviewers. (And for everyone who takes this file as a template.)